src/pages/contact.tsx

```diff
@@ -1,69 +1,73 @@
 import React from 'react';
-
-import Layout from '../components/layout/layout';
+import { Trans, useTranslation } from 'gatsby-plugin-react-i18next';
 import SEO from '../components/seo';
-import { SubTitle, Text, TextLink } from '../components/typography/typography';
-import { Grid, GridItem } from '../components/grid/grid';
-import { ContactForm } from '../components/form/contact';
-import styled from 'styled-components';
-import { up } from '../components/breakpoint/breakpoint';
+import { LayoutV2 } from '../components/layout/layout-v2';
 import { Leaflet } from '../components/leaflet/leaflet';
+import { SectionHeader } from '../new-components/section-header/section-header';
+import { Link } from '../components/links/links';
 import { graphql } from 'gatsby';
-import { Trans, useTranslation } from 'gatsby-plugin-react-i18next';
+import { ContentBlockContainer } from '../components/layout/content-block-container';
+import { ContactForm } from '../components/form/contact';
+import styled from 'styled-components';
+import { theme } from '../components/layout/theme';
 
-const ContactTitle = styled(SubTitle)`
-  margin-top: 40px;
+const StyledLink = styled(Link)`
+  color: ${theme.palette.text.link.default};
 
-  ${up('md')} {
-    margin-top: 120px;
+  &:hover {
+    border-bottom: 1px solid ${theme.palette.text.link.default};
   }
 `;
 
 interface ContactPageProps {
   location: Location;
 }
 
-const ContactPage: React.FC<ContactPageProps> = (props: ContactPageProps) => {
+const ContactPage = ({ location }: ContactPageProps) => {
   const { t } = useTranslation();
+
   return (
     <>
-      <Layout transparentHeader={true} hero={<Leaflet />}>
-        <SEO
-          title={`${t('contact.title')} | Satellytes`}
-          description={t('contact.info')}
-          location={props.location}
-        />
+      <SEO
+        title={`${t('contact.title')} | Satellytes`}
+        description={t('contact.info')}
+        location={location}
+      />
+      <LayoutV2 transparentHeader={true} light={true} hero={<Leaflet />}>
+        <ContentBlockContainer>
+          <SectionHeader
+            headline={t('contact.address')}
+            kicker={t('contact.title')}
+          >
+            Satellytes Digital Consulting GmbH
+            <br />
+            Sendlinger Straße 52
+            <br />
+            80331 München
+            <br />
+            <br />
+            <StyledLink to="https://goo.gl/maps/EGTh9xqgR7P871aC9">
+              Google Maps &gt;
+            </StyledLink>
+          </SectionHeader>
+        </ContentBlockContainer>
+
+        <ContentBlockContainer>
+          <SectionHeader headline={'E-Mail'}>
+            <Trans i18nKey="contact.info-link">
+              <p>
+                Nutzen Sie unser Kontaktformular oder schreiben Sie uns eine
+                E-Mail an
+                <StyledLink to="mailto:beep@satellytes.com">
+                  beep@satellytes.com
+                </StyledLink>
```

Comment:
Let's replace that mail with our generic one. We are more serious now than back in the days.

```suggestion
                <StyledLink to="mailto:info@satellytes.com">
                  info@satellytes.com
                </StyledLink>
```